grafana-plugin/src/containers/TeamsList/TeamsList.tsx

```diff
@@ -170,7 +170,7 @@ const TeamModal = ({ teamId, onHide }: TeamModalProps) => {
                   { label: 'Team members and admins', value: '0' },
                 ]}
                 value={shareResourceToAll}
-                onChange={setShareResourceToAll}
+                onChange={setShareResourceToAll as (res: string) => void}
```

Comment:
can we don't use type casting here?

Reply:
the setter is a React.Dispatch, while the `onChange` expects something else